_docs/runtime/git-sources.md

```diff
@@ -1,8 +1,38 @@
 ---
-title: "Managing your Git Sources"
+title: "DRAFT: Git Source management"
 description: ""
 group: runtime
 toc: true
 ---
 
-Coming soon
+A Git Source in CSDP is an ArgoCD application that references a Git repository. CSDP uses Git Sources to sync Argo Workflow and Argo Events resources to the cluster. 
+Git Sources are always associated with a specific runtime. A single runtime can have multiple Git Sources.
+
+When you provision a Codefresh runtime, a default Git Source is automatically created. This Git Source houses the Workflow Templates and Sensors. 
```

Comment:
it also houses other argo events resources like eventsources, i would add 'etc..' because there are others